packages/web-ui/src/ds/molecules/ChatWrapper/ChatTextArea/ToolCallForm/index.tsx

```diff
@@ -1,6 +1,10 @@
 'use client'
 import { MouseEvent, lazy, useState, useCallback, useMemo } from 'react'
-import { ToolMessage, Message, ToolCall } from '@latitude-data/compiler'
+import {
+  ToolMessage,
+  Message,
+  ToolCall,
+} from '@latitude-data/constants/legacyCompiler'
```

Comment:
Why we keep using legacy compiler? Why not use `promptl`?

Reply:
it's a major refactor, will do in the future